prow/config.yaml

```diff
@@ -268,6 +268,11 @@ presets:
           secretKeyRef:
             name: nightly-aks-github-integration-app-client-secret
             key: client-secret
+  - labels:
+      preset-kyma-keyring: "true"
+    env:
+      - name: KYMA_KEYRING
+        value: "kyma-prow"
```

Comment:
You don't use this anywhere, I think it can be removed?

Reply:
this is what I get for using master, this is part of another pr not yet merged